contracts/Storage.sol

```diff
@@ -93,4 +93,13 @@ abstract contract Storage is Constants {
     mapping(address => address) internal pTokenLookup; // PToken => underlying
     mapping(address => address) internal reversePTokenLookup; // underlying => PToken
     mapping(address => address) internal chainlinkPriceFeedLookup; // underlying => chainlinkAggregator
+
+    struct OverrideConfig {
+        bool enabled;
```

Comment:
That's a bit different though, we can disable CF this way. Also original design by Doug ;)

Reply:
hm, but if you make `override.collateralFactor == 0`, it will behave the same as `override.enabled == false && override.collateralFactor != 0` making `enabled` flag redundant. or is there any case where we'd like to have `override.enabled == true && override.collateralFactor == 0`?

Reply:
I think that was the initial idea. Could be useful in BB

Reply:
My original thinking was that enabled is necessary so that a CF can be explicitly set to 0 (without another special sentinel value that represents 0). I don't know if it's a valid use-case now that we have pause functionality, but this in theory *does* let you stop borrowing for a particular pair.

Reply:
Yeah in that case then it's OK to remove the `enabled` and let `collateralFactor=0` imply disabled. I'll let @dglowinski make that decision.

Reply:
governance could use it to help out a BB in distress possibly :)

Reply:
don't forget that we'll have asset policies so in distress any action can be paused for a given market. no need to configure it at the overrides level :)

Reply:
CF=0 override would actually prevent using regular BF and CF, **but** if liability is not borrow isolated, this could easily be side stepped by borrowing a second asset, in which case overrides would just be ignored. Still, let's decide on BB behaviour first before making the final call.

Reply:
oh, and also with CF=0 we can disable self-collateral on an asset, and it will be effective, since there's no self-collateral in cross borrows.